### PR TITLE
Updated eventuate util. Reconfigured swagger.

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -5,8 +5,8 @@ deployUrl=file:///Users/cer/.m2/testdeploy
 eventuateMavenRepoUrl=https://dl.bintray.com/eventuateio-oss/eventuate-maven-release/,file:///Users/cer/.m2/testdeploy,https://dl.bintray.com/eventuateio-oss/eventuate-maven-rc,https://snapshots.repositories.eventuate.io/repository
 
 springBootVersion=2.2.6.RELEASE
-eventuateTramVersion=0.24.0.BUILD-SNAPSHOT
-eventuateUtilVersion=0.2.0.RELEASE
+eventuateTramVersion=0.24.0.SNAPSHOT
```
Comment (cer) [inline gradle.properties:8]: .SNAPSHOT -> .BUILD-SNAPSHOT

File: gradle.properties [hunk 2]
```diff
@@ -5,8 +5,8 @@ deployUrl=file:///Users/cer/.m2/testdeploy
...
+eventuateUtilVersion=0.3.0.SNAPSHOT
```
Comment (cer) [inline gradle.properties:9]: 0.4.0.BUILD-SNAPSHOT

File: gradle.properties (hunk 1 above)
Comment (dartartem) [reply gradle.properties]: FIXED

File: gradle.properties (hunk 2 above)
Comment (dartartem) [reply gradle.properties]: FIXED

**Pull request merged**